std/conv.d

```diff
@@ -2341,13 +2341,15 @@ in
 body
 {
     import core.checkedint : mulu, addu;
+    import std.exception : enforce;
+
+    enforce!ConvException(!s.empty, "s must not be empty in integral parse");
```

Comment:
So when `parse` is called, the check is done twice. I suggest moving the check back to where it was before.